.github/workflows/push.yml

```diff
@@ -80,3 +80,5 @@ jobs:
             -d zend_extension=opcache.so
             -d opcache.protect_memory=1
             -d opcache.jit_buffer_size=16M
+      - name: Diff generated files
+        run: git add . -N && git diff --exit-code
```

Comment:
please advise the best place, should be run after `./.github/actions/test-linux`, macos, win, ...

Reply:
Location is fine. Linux is fine for now. We can do Windows once migrated to GitHub actions (if it even builds any more extensions).